Makefile

```diff
@@ -235,6 +235,19 @@ install-policy:
 	bin/iop ${ISTIO_NS} istio-policy ${BASE}/istio-policy --set global.istioNamespace=${ISTIO_NS} ${IOP_OPTS}
 	kubectl wait deployments istio-policy -n ${ISTIO_NS} --for=condition=available --timeout=${WAIT_TIMEOUT}
 
+# This target should only be used in situations in which the prom operator has not already been installed in a cluster.
+install-prometheus-operator:
+	# installs the prom operator in the default namespace
+	kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml
+	kubectl wait --for=condition=available --timeout=${WAIT_TIMEOUT} deploy/prometheus-operator
+
+# This target expects that the prometheus operator (and its CRDs have already been installed).
+# It is provided as a way to install Istio prometheus operator config in isolation.
+install-prometheus-operator-config:
+	kubectl create ns ${ISTIO_NS} || /bin/true
```

Comment:
can you change to `true` instead of `/bin/true`? Needed for mac compatibility apparently